xinference/api/restful_api.py

```diff
@@ -60,11 +63,28 @@
     top_k_field,
     top_p_field,
 )
-from ..types import ChatCompletion, Completion, CreateCompletion, Embedding, ImageList
+from ..types import ChatCompletion, Completion, CreateCompletion, ImageList
 
 logger = logging.getLogger(__name__)
 
 
+class JSONResponse(Response):
+    media_type = "application/json"
```

Comment:
Why this default value not set to the `__init__` method?

Reply:
This class is similar to the `JSONResponse` in `starlette/responses.py`, setting the `media_type` here is more clear.

Reply:
> This class is similar to the `JSONResponse` in `starlette/responses.py`, setting the `media_type` here is more clear.

Should we disable the `media_type` parameter in the `__init__` function? Because JsonResponse must be `application/json`.

Reply:
I changed the JSONResponse to inherit from starlette JSONResponse